internal/info/flipt.go

```diff
@@ -3,19 +3,27 @@ package info
 import (
 	"encoding/json"
 	"net/http"
+
+	"go.flipt.io/flipt/internal/config"
 )
 
 type Flipt struct {
```

Comment:
we have an opportunity to refactor this info API endpoint now I think?

maybe we should make the response it a bit more structured, something like:

```
{
  version: {
    current:
    latest: {
      version:
      URL:
    },
   build: {
    date: ,
    commit:
    goVersion:
    release:
   },
   authentication: {
    required: 
   },
   analytics: {
     enabled:
   },
   storage: {
    ...
  },
}
```

this would be an API response breaking change, but since the API is not documented for this endpoint maybe its ok?

wdyt?

Reply:
@markphelps  I’ve updated the new fields to match the desired format. However, I'n not ready to update old fields in this PR, as it would introduce too many changes at once.

Reply:
That sounds good to me